variables.tf

```diff
@@ -0,0 +1,87 @@
+variable "enabled" {
+  description = "Whether or not to create the IAM account alias"
+  default     = "true"
+}
+
+variable "name" {
+  description = "The Name of the account or solution  (e.g. `account`)"
+  default     = "account"
+}
+
+variable "namespace" {
+  description = "Namespace (e.g. `cp` or `cloudposse`)"
+  default     = ""
+}
+
+variable "stage" {
+  description = "Stage (e.g. `prod`, `dev`, `staging`)"
+  default     = ""
```

Comment:
remove `default` to make it required?

Reply:
https://github.com/cloudposse/terraform-aws-iam-account-settings/pull/1#discussion_r227531421